taxonomy.xml

```diff
@@ -1526,6 +1526,9 @@
          <category xml:id="Prayers" corresp="https://betamasaheft.eu/authority-files/Prayers/main">
             <catDesc>Prayers</catDesc>
          </category>
+         <category xml:id="Qene" corresp="https://betamasaheft.eu/authority-files/Qene/main">
+            <catDesc>Prayers</catDesc>
```

Comment:
oops, I merged too quickly, but noticed only now that the catDesc has to be updated ;) can you please create a new PR? thanks!